docs/wiki/[Examples]-Deploy-Connectivity-Resources.md

```diff
@@ -1 +1,165 @@
-_coming soon_
\ No newline at end of file
+## Overview
+
+This page describes how to deploy Enterprise-scale with the [Connectivity resources][wiki_connectivity_resources] created in the current Subscription context, using the default configuration settings.
+
+As connectivity resources can start to significantly increase Azure consumption costs, the module defaults are aimed to help build the basic connectivity configuration whilst minimising cost.
+Please refer to the Cloud Adoption Framework [Network topology and connectivity][ESLZ-Connectivity] recommendations to better understand which of these settings you should enable in a Production environment.
+
+In this example, we take a default configuration and make the following changes:
+
+- Set `deploy_connectivity_resources` to enable creation of the default Connectivity resources, including:
+  - Resource Group to contain all Connectivity resources.
+  - Virtual Network to use as a hub for hybrid-connectivity.
+  - Azure Private DNS Zones for Private Endpoints.
+  - Recommended Solutions for Azure Monitor.
+- Set the `subscription_id_connectivity` value to ensure the Subscription is moved to the correct Management Group, and policies are updated with the correct values.
+
+The module updates the `parameters` and `enforcement_mode` for a number of Policy Assignments, to enable features relating to the DDoS Protection Plan and Private DNS Zones for Private Endpoints.
+
+<!-- Some Private DNS Zones for Private Endpoints are bound to a specific Azure Region.
```

Comment:
Again an example here would help

Reply:
Do you mean within this comment? Or actually on the Wiki page?